spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+describe UsersController, type: :controller do
+
+  describe 'User creation' do
+    let(:admin) {create(:user, :admin)}
+    let(:hacker) {create(:user)}
+
+    context 'logged in as an admin' do
+      before :each do
+        allow(controller).to receive(:current_user) { admin }
+      end
+
+      it 'is able to create a user' do
+        expect{
+          post :create, user: attributes_for(:user)
+        }.to change(User, :count).by(2)
```

Comment:
oh is it because creating an admin counts towards the `User.count`?  I hate when it does that.